rs/bitcoin/validation/src/header.rs

```diff
@@ -262,7 +183,8 @@ fn compute_next_difficulty(
     // returned Regtest network doesn't adjust PoW difficult levels. For
     // regtest, simply return the previous difficulty target
 
-    if (prev_height + 1) % DIFFICULTY_ADJUSTMENT_INTERVAL != 0 || no_pow_retargeting(network) {
+    let height = prev_height + 1;
+    if height % DIFFICULTY_ADJUSTMENT_INTERVAL != 0 || no_pow_retargeting(network) {
```

Comment:
nit: I would have kept it as is. There are comments that reference the height value below that I assume need to be adjusted.

Reply:
done